src/nanoemoji/svg.py

```diff
@@ -83,24 +86,36 @@ def _add_unique_gradients(
             svg_defs.append(gradient)
 
 
-def _inter_glyph_reuse_key(painted_layer: PaintedLayer):
+# https://docs.microsoft.com/en-us/typography/opentype/spec/svg#coordinate-systems-and-glyph-metrics
+def _svg_matrix(transform: Affine2D) -> str:
+    # TODO handle rotation: units of Affine2D and svg matrix don't match
```

Comment:
I'm confused by this TODO. Why do you mean by "handle rotation: units of Affine2D and svg matrix don't match" here?

Reply:
Updated to explicitly note radians vs degrees